test/builder/lib/QuarkBuilderTest.sol

```diff
@@ -102,18 +103,28 @@ contract QuarkBuilderTest {
             morphoPositions: emptyMorphoPositions_(),
             morphoVaultPositions: emptyMorphoVaultPositions_()
         });
+        Accounts.QuarkSecret[] memory quarkSecrets = new Accounts.QuarkSecret[](3);
+        quarkSecrets[0] = quarkSecret_(address(0xb0b), BOB_DEFAULT_SECRET);
+        quarkSecrets[1] = quarkSecret_(address(0xa11ce), ALICE_DEFAULT_SECRET);
+        quarkSecrets[2] = quarkSecret_(address(0xfe11a), FELLA_DEFAULT_SECRET);
+
+        address[] memory accounts = new address[](3);
+        accounts[0] = address(0xb0b);
+        accounts[1] = address(0xa11ce);
+        accounts[2] = address(0xfe11a);
```

Comment:
Why `0xfe11a` instead of `0xc0b`? I thought you got rid of all references to `0xfe11a` in your test cases.

Reply:
Too many fella so I decided to just add fella's address 😄

Reply:
I just searched all `fella` and changed all to `cob`  no more random fella in house haha.